site/pages/rest-api/examples/create-role.md

```diff
@@ -4,13 +4,11 @@ title: Create Role
 
 This command will POST the file `role.xml` to the server to create a new role.
 
-[Sample Role XML File](./role.xml)
-
 # python
 ```python
 import requests
 with open('role.xml') as xml:
```

Comment:
let's try add the 'rb' flag to open the files in binary mode. This is important for all platforms to treat endlines the same way